metrics/report/makereport.sh

```diff
@@ -22,6 +22,11 @@ HOSTOUTPUTDIR="${SCRIPT_PATH}/output"
 GUESTINPUTDIR="/inputdir/"
 GUESTOUTPUTDIR="/outputdir/"
 
+# If in debugging mode, we also map in the scripts dir so you can
+# dynamically edit and re-load them at the R prompt
+HOSTSCRIPTDIR="${SCRIPT_PATH}/report_dockerfile"
```

Comment:
Nit: Maybe add some underscores to make reading this easier (`HOST_SCRIPT_DIR` is easier on my eyes ;)

Reply:
added the `_`s in an additional commit @jodh-intel , to try and allow the diffs to be just slightly easier to read.

`"`'d the 'live' as well...